pages/events/summer-school.vue

```diff
@@ -0,0 +1,288 @@
+          property: 'og:image:secure_url',
+          content: image
+        },
+        {
+          hid: 'og:image:alt',
+          property: 'og:image:alt',
+          content: title
+        }
+      ]
+    }
+  },
+  async asyncData (context: Context) {
+    const eventData = await context.$content('events/summer-school-2021').fetch() as IContentDocument
+
+    const headerRouteText = `${context.route.name} > header >`
+    const helpfulResourcesRouteText = `${context.route.name} > helpful-resources >`
+    eventData.header.card.segment = { action: `${headerRouteText} learn-more` }
+    eventData.header.cta.segment = { action: `${headerRouteText} qgss2021-enquiry-form` }
+    eventData.helpfulResources.resources[0].cta.segment = { action: `${helpfulResourcesRouteText} qgss2021-enquiry-form` }
+    eventData.helpfulResources.resources[1].cta.segment = { action: `${helpfulResourcesRouteText} install-here` }
+    eventData.helpfulResources.resources[2].cta.segment = { action: `${helpfulResourcesRouteText} qiskit-medium` }
+    eventData.helpfulResources.resources[3].cta.segment = { action: `${helpfulResourcesRouteText} code-of-conduct` }
+
+    eventData.mosaic.first.position = 'first'
+    eventData.mosaic.second.position = 'second'
+    eventData.mosaic.third.position = 'third'
+    eventData.mosaic.fourth.position = 'fourth'
+    eventData.mosaic.elements = [eventData.mosaic.first, eventData.mosaic.second, eventData.mosaic.third, eventData.mosaic.fourth]
+
+    const scheduleToTableData = (slot: AgendaSlot) => ([
+      {
+        component: 'span',
+        styles: 'min-width: 10rem; display: inline-block; font-weight: bold;',
+        data: slot.day
+      },
+      {
+        component: 'span',
+        styles: 'min-width: 10rem; display: inline-block; padding-top: 8px; padding-bottom: 8px',
+        data: slot.topic
+      },
+      {
+        component: 'span',
+        styles: 'min-width: 10rem; display: inline-block; padding-top: 8px; padding-bottom: 8px',
+        data: slot.speaker
+      },
+      {
+        component: 'span',
+        styles: 'min-width: 10rem; display: inline-block; padding-top: 8px; padding-bottom: 8px',
+        data: slot.format
+      }
+    ])
+
+    eventData.agenda.week1.tableData = eventData.agenda.week1.schedule.map(scheduleToTableData)
+    eventData.agenda.week2.tableData = eventData.agenda.week2.schedule.map(scheduleToTableData)
+
+    return {
+      eventData
+    }
+  }
+})
+export default class SummerSchoolPage extends QiskitPage {
+  routeName = 'summer-school'
+  agendaColumnsDataTable: string[] = ['Day', 'Topic', 'Speaker', 'Format']
+}
+</script>
+
+<style lang="scss" scoped>
+.summer-school-page {
+  color: $white-text-01;
+
+  &__section {
+    @include contained();
+    margin-top: $layout-05;
+    margin-bottom: $layout-03;
+
+    @include mq($until: large) {
+      margin-bottom: $layout-01;
+    }
+  }
+}
+</style>
+
+<style lang="scss">
+// overrides
```

Comment:
Please consider using a [deep selector](https://vue-loader.vuejs.org/guide/scoped-css.html#deep-selectors) to still scope these definitions to this component.

Reply:
@eddybrando - I've tried using deep selection before and was guided to not using them... 

https://github.com/Qiskit/qiskit.org/pull/951#discussion_r495666584


Unless I'm misunderstanding

Reply:
Oh ok, I get what is meant on that comment.

We should in general, when customising Carbon components, try and check if the customisation can be applied globally by updating the Carbon theme variables.

If that's not possible, then we should try and apply the styles globally, like those in the "checkbox", not from this specific component, but putting them on another file inside the **assets** directory.The reason being that

Reply:
I think that deep is not bad. What Salva said is to put that code into the community-theme.scss so you can reuse it in every component without a copy/paste.
We should talk with Russell to unify or rethink the use of the accordion and tabs.

Reply:
Pls let me know if this is right... 

or if you'd rather have me move these to the community-theme

Reply:
For me is a new refactor issue in collaboration with Russell.